deploy/Makefile

```diff
@@ -1,21 +1,50 @@
 ENV_FILE=.env
 include $(ENV_FILE)
 
-DOCKER_REMOTE=ssh://$(ZEN_WEB_REMOTE)
-PROJECT_NAME=harmony
+DOCKER_HOST ?= ssh://$(ZEN_WEB_REMOTE)
+PROJECT_NAME ?= harmony-web
 
-web_configure:
+configure:
 	@scp nginx/nginx_vhost_default_location $(ZEN_WEB_REMOTE):${NGINX_DEFAULT_VHOST_CONFIG}
 	@scp instance_config.json $(ZEN_WEB_REMOTE):${INSTANCE_CONFIG}
 	@scp global_config.py $(ZEN_WEB_REMOTE):${GLOBAL_CONFIG}
 
-web_deploy:
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml up -d --force-recreate
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker system prune -af
+prune:
+	@DOCKER_HOST=$(DOCKER_HOST) docker system prune -af
```

Comment:
Moved prune to it's own command - you don't always want to prune.

Reply:
Maybe you can explain to me why you think this step is required? I don't get why you'd want to prune after starting/stopping the containers?

Reply:
This cleans old images for the most part and keeps docker lean. I've run into this a lot where machine space runs out etc due to docker taking it all.

More relevant to dev instances, but prod should also be considered.

Reply:
If we only had one docker-compose I'd agree - but what happens is that you stop web, after having stopped druid, and it deletes all your druid images. Very frustrating.

That's why I'd rather have it be an explicit, intentional action, than a side effect of starting up web.

Reply:
Fair enough, think this is a spill over on what we do in fab scrips 👀

We'd just want to note on this, repeated deployments will most likely result in soace running out.

This to me is really not a big deal, but you are going to use it way more than me so full speed ahead!

Reply:
Oh - I like the filter! I still don't think it should be doing it as a side effect, but `--filter "until=24h"` will actually resolve what's been bothering me, so it's a fine compromise.